desktop/mac/permission-requirements.md

```diff
@@ -14,34 +14,99 @@ It also provides clarity on running containers as `root` as opposed to having `r
 ## Permission requirements
 
 Docker Desktop for Mac is run as an unprivileged user. However, certain functionalities may be required for Docker Desktop to perform a limited set of privileged configurations such as:
- - [Installing symlinks](#installing-symlinks) in `/usr/local/bin`. This ensures the `docker` CLI is on the user’s PATH without having to reconfigure shells, log out then log back in, for example.
+ - [Installing symlinks](#installing-symlinks) in`/usr/local/bin`. 
  - [Binding privileged ports](#binding-privileged-ports) that are less than 1024. The so-called "privileged ports" have not generally been used as a security boundary, however OSes still prevent unprivileged processes from binding them which breaks commands like `docker run -p 127.0.0.1:80:80 docker/getting-started`.
  - [Ensuring `localhost` and `kubernetes.docker.internal` are defined](#ensuring-localhost-and-kubernetesdockerinternal-are-defined) in `/etc/hosts`. Some old macOS installs did not have `localhost` in `/etc/hosts`, which caused Docker to fail. Defining the DNS name `kubernetes.docker.internal` allows us to share Kubernetes contexts with containers.
  - Securely caching the Registry Access Management policy which is read-only for the developer.
 
+Depending on which version of Docker Desktop for Mac is used, privileged access is granted either during installation, first run, or only when it is needed.
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#tab1">Version 4.18 and later</a></li>
+  <li><a data-toggle="tab" data-target="#tab2">Version 4.15 - 4.17 </a></li>
+  <li><a data-toggle="tab" data-target="#tab3">Versions prior to 4.15</a></li>
+</ul>
+<div class="tab-content">
+<div id="tab1" class="tab-pane fade in active" markdown="1">
+
+From version 4.18 and above of Docker Desktop, privileged access for certain functionalities is granted during installation. The first time Docker Desktop for Mac is launched, the user is presented with an installation window to either use the recommended settings, or configure:
+- The location of the Docker CLI tools either in the system or user directory
+- The default Docker socket
+- Privileged port mapping. 
+
+The user is then asked for their password to confirm. 
```

Comment:
The user is not asked for their password if they change some of the Advanced settings. That's the point of this new flow, so we should adjust this.